src/main/java/com/thirdcc/webapp/service/impl/TransactionServiceImpl.java

```diff
@@ -86,4 +102,19 @@ public void delete(Long id) {
         log.debug("Request to delete Transaction : {}", id);
         transactionRepository.deleteById(id);
     }
+
+    @Override
+    public Page<TransactionDTO> findAllByEventId(Long eventId, Pageable pageable) {
+        log.debug("Request to find all Transaction of event: {]", eventId);
```

Comment:
got typo in log debug, the close bracket is } not ]

And for this pr is not checking whether the user is event crew or admin or not right?

Reply:
resolve in 6b9182207f9785d5608b022dfdf8e229b708f74d